ibus: Don't run dbus tests within the build.

  Prevent a test requiring spawning a dbus session from running
  during the build.

  Upstream-status: Inappropriate [disable-feature]

  Signed-off-by: Brendan Kerrigan <kerriganb@ainfosec.com>   

diff --git a/data/dconf/make-dconf-override-db.sh b/data/dconf/make-dconf-override-db.sh
index 9c650e9..d7320c3 100755
--- a/data/dconf/make-dconf-override-db.sh
+++ b/data/dconf/make-dconf-override-db.sh
@@ -12,10 +12,6 @@ export XDG_CACHE_HOME="$TMPDIR/cache"
 export GSETTINGS_SCHEMA_DIR="$TMPDIR/schemas"
 mkdir -p $XDG_CONFIG_HOME $XDG_CACHE_HOME $GSETTINGS_SCHEMA_DIR
 
-eval `dbus-launch --sh-syntax`
-
-trap 'rm -rf $TMPDIR; kill $DBUS_SESSION_BUS_PID' ERR
-
 # in case that schema is not installed on the system
 glib-compile-schemas --targetdir "$GSETTINGS_SCHEMA_DIR" "$PWD"
 
@@ -47,4 +43,3 @@ done
 
 rm -rf $TMPDIR
 
-kill $DBUS_SESSION_BUS_PID
